packages/typespec-test/test/anomalyDetector/generated/typespec-ts/review/ai-anomaly-detector.api.md

```diff
@@ -359,7 +359,7 @@ export interface ListMultivariateModelsDefaultHeaders {
 // @public (undocumented)
 export interface ListMultivariateModelsDefaultResponse extends HttpResponse {
     // (undocumented)
-    body: MultivariateErrorResponseOutput;
+    body: MultivariateResponseErrorOutput;
```

Comment:
Can you check if we still keep the original `MultivariateErrorResponseOutput` model ? I don't see where it's being deleted ?